docs/casa/plugins/cert-authn-tutorial.md

```
@@ -0,0 +1,97 @@
+
+1. Still in the Firefox settings, locate "Security devices" and click on "Load". Choose a name for the Yubikey PKCS#11  module and browse to the `bin` directory of the PIV tool. Finally select the file `libykcs11.dll`
+
+1. Restart Firefox
+
+### Test
+
+Using Firefox, follow steps similar to those when the plugin was formerly [tested](./cert-authn.md#testing). Here, the browser dialog for picking a cert will be shown, and then a prompt will appear for entering the Yubikey PIN.
+
+## Useful resources
+
+- https://support.yubico.com/s/article/YubiKey-smart-card-deployment-guide
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)